mage_integrations/mage_integrations/destinations/opensearch/__init__.py

```diff
@@ -15,6 +28,28 @@ def _process(self, input_buffer) -> None:
             file_input=open(self.input_file_path, 'r')
         )
 
+    def test_connection(self) -> None:
+        config = {}
```

Comment:
Could you not duplicate the code? You can call the method here: https://github.com/mage-ai/mage-ai/blob/master/mage_integrations/mage_integrations/destinations/opensearch/target_opensearch/sinks.py#L181-L198